### DISCUSSION TITLE
What are the complete command line options?

### DISCUSSION THREAD

**Discussion by carco5a** [Sanster/IOPaint#447] [Q&A] [answered]
--help just shows the high-level items. How do I see the full list?
Mainly I just want to know what replaced --load-installer-config from lama

**Comment by Sanster** [comment, marked as answer]
You can you use `iopaint start --help` to see options for `start` command.

Using `iopaint start --config config.json` to start server using a config file.

**Reply by carco5a** [reply]
Ah, of course -- I see now, thank you. Have fun with BG3.
